contrib/babelfishpg_tsql/runtime/functions.c

```diff
@@ -573,7 +573,8 @@ schema_id(PG_FUNCTION_ARGS)
 
 	id = get_namespace_oid(physical_name, true);
 
-	pfree(name);
+	if (!name)
```

Comment:
Shouldn't this be `if(name)` instead of `if(!name)`?

Reply:
Addressed.